phoenix-core/src/test/java/org/apache/phoenix/query/BaseTest.java

```diff
@@ -1692,7 +1697,57 @@ public Void call() throws Exception {
 
         LOGGER.info("Disabled and dropped {} tables in {} ms", tableCount, endTime-startTime);
     }
-    
+
+    protected int setBindVariables(PhoenixPreparedStatement stmt, int startBindIndex, int numBinds, PDataType[] testPKTypes)
+            throws SQLException {
+
```

Comment:
Should this be in BaseTest? It seems to make some test-specific assumptions (like defaulting a VARCHAR(25), and the random limits.